WordPress/Classes/Stores/RemoteFeatureFlagStore.swift

```diff
@@ -3,23 +3,25 @@ import WordPressKit
 
 class RemoteFeatureFlagStore {
 
-    /// Alllow injecting the FeatureFlagRemote for tests (or to customize the WordPress.com API)
-    private let remote: FeatureFlagRemote
+    public static let shared = RemoteFeatureFlagStore()
 
-    /// Create a new RemoteFeatureFlagStore, with an optionally injected FeatureFlagRemote. If none is provided, it will use  an anonymous connection to WordPress.com.
-    ///
-    /// - Parameters:
-    ///     - remote: An optional FeatureFlagRemote with a default WordPressComRestApi instance. Inject a FeatureFlagRemote with a different WordPressComRestApi instance
-    ///     to authenticate with the Remote Feature Flags endpoint – this alllows customizing flags server-side on a per-user basis.
-    init(remote: FeatureFlagRemote = FeatureFlagRemote(wordPressComRestApi: WordPressComRestApi.defaultApi())) {
-        self.remote = remote
+    private init() {
```

Comment:
One unfortunate aspect of the original design of this object is that it can't take injected `UserDefaults` (or equivalent).

Given the increased reliance on it, I'd suggest something like:

```suggestion

private let userDefaults: UserPersistentRepository

init(userDefaults: UserPersistentRepository = UserDefaults.standard ) {
  self.userDefaults = userDefaults
}
```

Reply:
Do you have scenarios in mind in which we'll need to inject a custom `UserDefaults` other than in tests?

Reply:
Not at the moment, though an in-memory `UserDefaults` implementation (without all the disk synchronization logic) is orders of magnitude faster, which keeps our test suite nice and fast.

Reply:
> orders of magnitude faster, which keeps our test suite nice and fast.

That's never a bad thing! Done in f37eb22285cbc6fac42a2022df5287309cf00014 👍